### ISSUE TITLE
Dualshock 4 v2 (model CUH-ZCT2) does not detected by game.

### ISSUE THREAD

**Issue by KainXVIII** [renaudbedard/fez-1.12-issues#169]
Is it only my problem?

**Comment by KainXVIII** [comment]
Its seems that new Dualshock 4 has different ID
Here is dxdiag.txt
https://drive.google.com/open?id=0B2Ihp2ntZp5MR1J0LUs0UkVUSEk

**Comment by flibitijibibo** [comment]
Yeah, the Slim has a new GUID so all games will have to update for this. I'll probably fix this when SDL 2.0.6 is out since some GUID changes are happening at the same time over there.

**Comment by KainXVIII** [comment]
Nice!

**Comment by KainXVIII** [comment]
I tried new copy&paste fresh gamecontrollerdb.txt from their github page - now game can be controlled with DS4, but still without playstation button prompts and vibration, interesting

**Comment by renaudbedard** [comment]
Fixed the PlayStation button prompts for this new model and the wireless adapter.
The vibration is not really up to me, depends whether SDL and/or the driver reports that it supports haptics. I'll update this when the patch is live and you can let me know if it works! :)

**Comment by KainXVIII** [comment]
Ok, thanks!
PS - is there some special ds4 lightbar support in Fez or its just glows cyan?

**Comment by flibitijibibo** [comment]
It's supported on Linux but not other platforms, mostly because there's no easy way to access the LEDs as a separate device. Third party software supports it but it's always tightly coupled to the rest of the program (DS4Windows, Steam Controller driver, etc).

**Comment by KainXVIII** [comment]
Same for vibration? Thanks for the info!
